static/styles/app_components.css

```diff
@@ -740,7 +740,14 @@ div.overlay {
     }
 }
 
-#settings_page {
+.save-button-controls {
```

Comment:
We can change this in a follow-up commit, but should this be called `save-discard-widget-controls`? The discard button seems to be inside this block.